RabbitMQ.Stream.Client/Reliable/ReliableProducer.cs

```diff
@@ -17,6 +17,7 @@ public record ReliableProducerConfig
     public Func<MessagesConfirmation, Task> ConfirmationHandler { get; init; }
     public string ClientProvidedName { get; set; } = "dotnet-stream-rproducer";
     public IReconnectStrategy ReconnectStrategy { get; set; } = new BackOffReconnectStrategy();
+    public TimeSpan TimeoutMessageAfter { get; init; } = TimeSpan.FromSeconds(3);
```

Comment:
Should we add a sort of validation to avoid to insert wrong values like `0` or `Hours` ?

Reply:
No idea - 0 doesn't make sense for sure, so we can definitely check for that. For other values - other than just wasting space in cache, what's the danger on waiting for days on end? :D

Reply:
> what's the danger on waiting for days on end? :

The unconfirmed list is in memory. Just a memory problem. I'd say!